tracer/src/Datadog.Trace/RemoteConfigurationManagement/Product.cs

```diff
@@ -42,6 +42,9 @@ public void AssignConfigs(List<RemoteConfiguration> changedConfigs)
                 }
             }
 
+            Log.Debug($"Received {changedConfigs.Count} Remote Configuration records for product {Name}, " +
+                      $"of which {filteredConfigs?.Count ?? 0} matched the predicate.");
+
```

Comment:
nit: Shouldn't use interpolated strings for logs where you're going to write the log more than once, as it messes with the template cache. Same goes for the other log messages too

Reply:
D'oh! I forgot about this :( Fixed in 8783284f81fabc53121b1c1a2d37507487718b80